src/dialogs/more-info/controls/more-info-timer.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  LitElement,
+  html,
+  TemplateResult,
+  CSSResult,
+  css,
+  property,
+  PropertyValues,
+} from "lit-element";
+import "@material/mwc-button";
+
+import { HomeAssistant } from "../../../types";
+
+class MoreInfoTimer extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() public stateObj?: any; // TODO
+
+  protected render(): TemplateResult | void {
+    if (!this.stateObj) {
+      return html``;
+    }
+
+    const actions = ["start", "pause", "cancel", "finish"];
+
+    return html`
+      <ha-attributes .stateObj="${this.stateObj}"></ha-attributes>
+      ${actions.map((state) => {
```

Comment:
We should hide/show buttons based on the current state.

So only show a cancel if it has been started before.  Only show pause if it is running etc.